packages/utils/src/network.ts

```diff
@@ -26,6 +26,7 @@ export function getBrowserOnlineStatus() {
 export async function getReactNativeOnlineStatus(): Promise<boolean> {
   // global.NetInfo is set in react-native-compat
   if (isReactNative() && typeof global !== "undefined" && (global as any)?.NetInfo) {
+    return true; // TODO: discuss on adding react-native-community/netinfo
```

Comment:
@matias-la `react-native-community/netinfo` gabriel/fix/wallet-connection is needed for this. what'd be best action here ?

Reply:
repace with `@exodus/netinfo` ?

Reply:
@ezenwankwogabriel yeah, we have `@exodus/netinfo@1.4.2` audited - could you please try it?